client/dive-common/apispec.ts

```diff
@@ -7,6 +7,7 @@ import { CustomStyle } from 'vue-media-annotator/use/useStyling';
 
 type DatasetType = 'image-sequence' | 'video';
 type MultiTrackRecord = Record<string, TrackData>;
+type SubType = 'stereo' | 'multicam' | null;
```

Comment:
Dataset is `image-sequence | video | multi` - I didn't add the `multi` to the  common/apispec because it isn't supported on the Web side yet.  I could add it but it would require some updating to other files outside of the desktop scope to support it.  Maybe I should place a comment here?

So I could see having DatasetType renamed to MediaType where it has `image-sequence | video | multi(desktop only)`  I think I would want to rename it on the Server backend in python as well at the same time (a lot more files touched).

SubType may more accurately be renamed as multiCamType?  There are differences in pipelines than can only be run based on the subType.  Stereo only pipelines vs Multicam only pipelines.  This allows me to disable/enable pipelines without having to reach far down into the meta and create a computed property (especially in the future folder view on web).  I could move this into camera object although we may want to use this same field for for future identification of specialized media outside of MultiCam subtypes.  Like IR images may have special pipelines on them and can be designated using this field.

The secondary use of it is inside of the JSONMeta for the displaying of the icon indicating that the dataset is stereo (binocular icon) or multicam (burst image icon).

So I can move it into the camera field and remove the icon display capability in recents because I wouldn't know the type until I load the fully metadata Or I could rename it to something like  `multiCamSubType` or `multiCamType`.